tests/qtest/libqos/virtio-video.c

```diff
@@ -0,0 +1,179 @@
+/*
+ * libqos virtio-video driver
+ *
+ * Copyright (c) 2023 Red Hat Inc.
+ * 
+ * Authors:
+ *  Albert Esteve <aesteve@redhat.com>
+ *   (based on virtio-gpio.c)
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#include "qemu/osdep.h"
+#include "standard-headers/linux/virtio_config.h"
+#include "../libqtest.h"
+#include "qemu/module.h"
+#include "qgraph.h"
+#include "virtio-video.h"
+
+#define NUM_VQUEUES 2
```

Comment:
I am wondering if this define should be in virtio-video.h. IIRC this was used also in the virtio-video code but hardcoding the '2', so maybe would be nice to have this in a include file and use it in all the code.

Reply:
Yes, but this is for the test library. In other files in the same folder it is actually hardcoded as a magic number too. Here I decided to define it to reuse it, and make it more readable. In virtio-video.h is only used once to assign it to `nvqs`... so I think I would leave it as is for the time being, both for consistency with surrounding code in the case of the test, and for simplicity in the case of the main device code.